.mergify.yml

```diff
@@ -0,0 +1,22 @@
+pull_request_rules:
+  - name: Merge approved and green PRs with `merge when green` label
+    conditions:
+      - "#approved-reviews-by>=1"
+      - status-success=test
```

Comment:
I checked the recent status checks and it looks like for Rust, it is called "build" for whatever arbitrary reason:

![image](https://user-images.githubusercontent.com/4210206/94153362-f9e29f80-fe7c-11ea-8dd3-55bab08b193a.png)

```suggestion
      - status-success=build
```